lazyflow/utility/helpers.py

```diff
@@ -127,3 +129,30 @@ def bigintprod(nums: Iterable[numbers.Integral]) -> int:
     array shapes seen in practice on some operating systems (windows: long -> 32 bit).
     """
     return reduce(mul, map(int, nums))
+
+
+def get_ram_per_element(dtype: Union[object, numpy.dtype]) -> int:
+    """Get number of bytes for a certain data type
+
+    Examples:
+    >>> get_ram_per_element(object)
+    16
+    >>> get_ram_per_element(numpy.float64)
+    8
+    >>> get_ram_per_element(numpy.float64())
+    8
+    >>> get_ram_per_element(numpy.uint8)
+    1
+    >>> get_ram_per_element(numpy.bool)
+    1
+    >>> get_ram_per_element(int)
+    8
+    """
+    dtype = numpy.dtype(dtype)
+
+    if numpy.issubdtype(dtype, numpy.number):
+        return dtype.itemsize
+    elif numpy.issubdtype(dtype, numpy.bool_):
+        return dtype.itemsize
+    else:
+        return sys.getsizeof(None)
```

Comment:
Why are we using the size of `None` here? Are we trying to guesstimate the required size of an object that is not a numpy number/boolean without creating an instance?

Reply:
I have really no clue (see commit comment: "in principle we don't need this - we could simply
`numpy.dtype(whatever).itemsize`. For some reason the old code treats `object`
and `numpy.object_` differently (numpy will give itemsize = 8 on my machine,
vs `sys.getsizeof(None) = 16). Keeping this behavior for now.")

Reply:
Hmm, it would be helpful to put this in a comment then. Commit messages are great, but I think this is quite obscure and should be more visible.